source/mir/lapack.d

```diff
@@ -336,6 +336,147 @@ size_t spev(T)(
 }
 
 ///
+size_t sytrf(T)(
+    Slice!(Canonical, [2], T*) a,
+    Slice!(Contiguous, [1], lapackint*) ipiv,
+    Slice!(Contiguous, [1], T*) work,
+    char uplo
+    )
+{
+    assert(a.length!0 == a.length!1, "mutrix must be squared");
```

Comment:
Typo mutrix -> matrix.
Also `squared` should be `square` (in all the messages), make the message an private enum since they are all the same.